backend/pkg/version/version.go

```diff
@@ -0,0 +1,60 @@
+package version
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"runtime"
+	"runtime/debug"
+	"text/tabwriter"
+)
+
+var (
+	version string = "v0.0.0-dev"
+)
+
+type Version struct {
+	Version      string
+	GoVersion    string
+	GitCommit    string `json:",omitempty"`
+	RevisionTime string `json:",omitempty"`
+	OS           string
+	Arch         string
+}
+
+func Get() Version {
+	v := Version{
+		Version:   version,
+		GoVersion: runtime.Version(),
+		OS:        runtime.GOOS,
+		Arch:      runtime.GOARCH,
+	}
+	if bldInfo, ok := debug.ReadBuildInfo(); ok {
+		for _, setting := range bldInfo.Settings {
+			switch setting.Key {
+			case "vcs.revision":
+				v.GitCommit = setting.Value
+			case "vcs.time":
+				v.RevisionTime = setting.Value
+			}
+		}
+	}
+	return v
+}
+
+func (v Version) String() string {
+	var buf bytes.Buffer
+	w := tabwriter.NewWriter(&buf, 0, 8, 0, '\t', 0)
+	fmt.Fprintf(w, "Version:\t%s\n", version)
```

Comment:
```suggestion
	fmt.Fprintf(w, "Version:\t%8s\n", version)
```
`8` is just an example -- but for cool alignment in column you can use it here.

Reply:
The `w` is a `tabwriter.Writer` which formats the information into a (left-aligned) table. The tab `\t` character is used as a separator, but may actually result in two tabs if the column is too wide.